### PR TITLE
 Split Nginx config into several files for easier overriding

### PR DESCRIPTION
**Pull request by JayH5**

### CODE REVIEW

File: nginx/nginx.conf [hunk 1]
```diff
@@ -83,5 +83,5 @@ http {
         text/css
         text/plain;
 
-    include /etc/nginx/conf.d/*.conf;
+    include conf.d/*.conf;
```
Comment (bearnard) [inline nginx/nginx.conf:86]: how does this account for `conf.d/django.conf.d/locations/static.conf` ?

Comment (bearnard) [reply nginx/nginx.conf:86]: ah, nm, found that line.

Comment (JayH5) [reply nginx/nginx.conf:86]: It doesn't it just includes `conf.d/django.conf` and that includes the things in `conf.d/django.conf.d`.

**Comment by bearnard** [comment]
👍 Apart from the unconventional nested conf.d thing, I agree, don't see a better way.

**Pull request merged**